code/game/mecha/combat/five_stars.dm

```diff
@@ -0,0 +1,22 @@
+/obj/mecha/combat/five_stars
+	desc = "A state of the art tank deployed by the Spinward Stellar Coalition National Guard."
+	name = "\improper Tank"
+	icon = 'icons/mecha/mecha_96x96.dmi'
+	icon_state = "five_stars"
+	armor = list("melee" = 100, "bullet" = 50, "laser" = 35, "energy" = 35, "bomb" = 0, "bio" = 0, "rad" = 0, "fire" = 100, "acid" = 100)
+	exit_delay = 40
+	step_in = 4
+	dir_in = 1 //Facing North.
+	max_integrity = 800
+	pixel_x = -32
```

Comment:
Set up proper multi-tile object support if you want this. Spacepods was closed because Mrty didnt want to add multi tile objects and its the same problem here

Reply:
I don't have the technical know-how to make lighting work with this. Have any of our downstreams added multi-tile objects?

Reply:
wait, i think i know how to do this

Reply:
Yog has something but it's coded by monster so i wouldnt take it as an example.

Reply:
That's incredibly rude.